pkg/tests/api/alerting/api_notification_channel_test.go

```diff
@@ -220,31 +250,60 @@ func (nc *mockNotificationChannel) matchesExpNotifications(exp map[string][]stri
 		}
 		for i := range expVals {
 			expVal := expVals[i]
-			var r *regexp.Regexp
+			var r1, r2 *regexp.Regexp
 			switch expKey {
 			case "webhook_recv/webhook_test":
 				// It has a time component "startsAt".
-				r = regexp.MustCompile(`.*"startsAt"\s*:\s*"([^"]+)"`)
+				r1 = regexp.MustCompile(`.*"startsAt"\s*:\s*"([^"]+)"`)
 			case "slack_recvX/slack_testX":
 				fallthrough
 			case "slack_recv1/slack_test_without_token":
 				// It has a time component "ts".
-				r = regexp.MustCompile(`.*"ts"\s*:\s*([0-9]{10})`)
+				r1 = regexp.MustCompile(`.*"ts"\s*:\s*([0-9]{10})`)
+			case "sensugo/events":
+				// It has a time component "ts".
+				r1 = regexp.MustCompile(`.*"issued"\s*:\s*([0-9]{10})`)
 			case "pagerduty_recvX/pagerduty_testX":
 				// It has a changing "source".
-				r = regexp.MustCompile(`.*"source"\s*:\s*"([^"]+)"`)
+				r1 = regexp.MustCompile(`.*"source"\s*:\s*"([^"]+)"`)
+			case "googlechat_recv/googlechat_test":
+				// "Grafana v | 25 May 21 17:44 IST"
+				r1 = regexp.MustCompile(`.*"text"\s*:\s*"(Grafana v[^"]+)"`)
+			case "victorops_recv/victorops_test":
+				// It has a time component "timestamp".
+				r1 = regexp.MustCompile(`.*"timestamp"\s*:\s*([0-9]{10})`)
+			case "v1/alerts":
+				// It has a changing time fields.
+				r1 = regexp.MustCompile(`.*"startsAt"\s*:\s*"([^"]+)"`)
+				r2 = regexp.MustCompile(`.*"UpdatedAt"\s*:\s*"([^"]+)"`)
```

Comment:
out of curiosity, why it's not `updatedAt`?

Reply:
No idea, it comes from Prometheus Alertmanager